common/lib/xmodule/xmodule/library_tools.py

```diff
@@ -137,11 +169,13 @@ def can_use_library_content(self, block):
 
     def update_children(self, dest_block, user_perms=None, version=None):
         """
-        This method is to be used when the library that a LibraryContentBlock
-        references has been updated. It will re-fetch all matching blocks from
-        the libraries, and copy them as children of dest_block. The children
-        will be given new block_ids, but the definition ID used should be the
-        exact same definition ID used in the library.
+        Update xBlock's children.
+
+        Re-fetch all matching blocks from the libraries, and copy them as children of dest_block.
+        The children will be given new block_ids.
```

Comment:
`The children will be given new block_ids.` --> `New block_ids will be given to the children`

Reply:
But they are equal))) I'll leave it as it is if you don't mind)